common/reviews/api/rush-lib.api.ts

```diff
@@ -496,19 +517,30 @@ class VersionMismatchFinder {
 class VersionPolicy {
   constructor(versionPolicyJson: IVersionPolicyJson);
   // (undocumented)
+  public abstract bump(bumpType?: BumpType, identifier?: string): void;
```

Comment:
>bumpType?: BumpType, identifier?: string [](start = 23, length = 40)

Why are these optional?  It would be better to make them required parameters that can be "string|undefined", to ensure that callers don't forget to pass along one of the parameters.